test/e2e/atlas/streams_test.go

```diff
@@ -275,13 +275,11 @@ func TestStreams(t *testing.T) {
 		}{
 			{Name: connectionName, Type: "Kafka", BootstrapServers: "example.com:8080,fraud.example.com:8000"},
 			// sample_stream_solar is a sample connection which is added by default to new stream instances
-			{Name: "sample_stream_solar", Type: "Sample", BootstrapServers: ""},
```

Comment:
created a ticket to support the flag that was added https://jira.mongodb.org/browse/CLOUDP-268860